src/Catel.MVVM/Catel.MVVM.Shared/MVVM/ViewModels/ViewModelBase.cs

```diff
@@ -92,6 +92,14 @@ public abstract partial class ViewModelBase : ValidatableModelBase, IRelationalV
 #endif
         private readonly bool _ignoreMultipleModelsWarning;
 
+        /// <summary>
+        /// Value indicating whether the identifier will be reused from the relased identifier pool.
+        /// </summary>
+#if NET
+        [field: NonSerialized]
+#endif
+        private readonly bool _reuseIdentifier;
```

Comment:
This field is not required, see comments later in this file.

Reply:
Fixed

Reply:
Fixed